packages/mds-audit/api.ts

```diff
@@ -226,6 +240,7 @@ function api(app: express.Express): express.Express {
 
             // 200 OK
             return res.status(200).send({
+              version: res.locals.version,
```

Comment:
Nit: could this be destructured combined with the other destructuring on line 191? Then we could use the es6 object literal shorthand `{ version }`